hs_core/page_processors.py

```diff
@@ -312,7 +323,8 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                'maps_key': maps_key,
                'topics_json': mark_safe(escapejs(json.dumps(topics))),
                'czo_user': any("CZO National" in x.name for x in user.uaccess.communities),
-               'odm2_terms': list(ODM2Variable.all())
+               'odm2_terms': list(ODM2Variable.all()),
+               'external_content': len([x.file_name for x in content_model.files.all() if '.url' in x.file_name])
```

Comment:
This is costly and not absolutely reliable in finding referenced content.  Referenced content is a SingleFile aggregation with extra_date key of url.  Looping through all files in the resource and looking for something that has .url in it could return some false positives.